File: reports/releases/v0.7.1/attempts/overhead-infer-c-attempt-01/outputs/reports/raw/invocation-overhead/infer-c/fixture/dfb_trivial.c
int dfb_source(void) { // DFB-SOURCE: trivial-overhead-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: trivial-overhead-sink

void run(void) {
    dfb_source();
    dfb_sink(0);
}

File: reports/releases/v0.7.1/attempts/overhead-joern-php-attempt-01/outputs/reports/raw/invocation-overhead/joern-php/fixture/dfb_trivial.php
<?php
function dfb_source(): string { // DFB-SOURCE: trivial-overhead-input
    return "tainted";
}

function dfb_sink(string $value): void {} // DFB-SINK: trivial-overhead-sink

function run(): void {
    dfb_source();
    dfb_sink("clean");
}
